pycbc/events/events.py

```diff
@@ -193,6 +193,20 @@ def newsnr(snr, reduced_x2, q=6., n=2.):
         return newsnr
     else:
         return newsnr[0]
+       
+def bluesnr(snr, bchisq, lchisq):
+    """ Combined SNR derived from NewSNR and Lat Chisq"""
```

Comment:
Should bluesnr be implemented as a new detection statistic, or should it be implemented as a cut (ie. something run at the end of pycbc_inspiral, controlled with a `--lat-chisq-cut = ??` option. That would not then tie the lat-chisq cut to newsnr.